internal/api/router.go

```diff
@@ -105,6 +110,9 @@ func NewRouter(handlers Handlers, cfg *config.Configuration, logger *logger.Logg
 		user := v1Private.Group("/users")
 		{
 			user.GET("/me", handlers.User.GetUserInfo)
+			user.POST("", handlers.User.CreateUser)
+			user.GET("/service-accounts", handlers.RBAC.ListServiceAccounts)
+			user.GET("/service-accounts/:id", handlers.RBAC.GetServiceAccount)
```

Comment:
`/search` API with types etc filter and these routes to be in user service not in RBAC handler